templates/terraform/encoders/monitoring_notification_channel.go.erb

```diff
@@ -0,0 +1,31 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+labelmap, ok := obj["labels"]
+if !ok {
+	labelmap = make(map[string]string)
+}
+
+var labels map[string]string
+labels = labelmap.(map[string]string)
+
+for _, sl := range sensitiveLabels {
```

Comment:
can we change this to expand on v, ok := d.GetOk("sensitive_labels")?

Reply:
If you are talking about line 23 it's just iterating through the constant slice of possible labels to check for.
If you are talking about line 24, are you wanting to update the variable names or GetOk vs GetOkExists?